ui.frontend/src/utils.js

```diff
@@ -219,8 +219,12 @@ class Utils {
      */
     static updateId(htmlElement, oldId, newId) {
         let elementWithId = htmlElement.querySelectorAll("#" + oldId)[0];
+        let elementWithErrorId = htmlElement.querySelector("#" + `${oldId}__errormessage`);
         if (elementWithId) {
             elementWithId.id = newId;
+        };
+        if (elementWithErrorId) {
```

Comment:
For repeated instances, you must update the value of the "aria-describedby" attribute.

Reply:
Added!